docs/Analyzer Configuration.md

```diff
@@ -94,6 +94,7 @@ Configurable Rules:
 [CA1802](https://docs.microsoft.com/visualstudio/code-quality/ca1802),
 [CA1815](https://docs.microsoft.com/visualstudio/code-quality/ca1815),
 [CA1819](https://docs.microsoft.com/visualstudio/code-quality/ca1819),
+[CA2208](https://docs.microsoft.com/visualstudio/code-quality/ca2208),
```

Comment:
This is unrelated, but don't want separate PR for just one row

Reply:
If this is really implemented the doc page isn't updated, could you please create a ticket?

Reply:
Sorry, i am not familiar how the doc page updated, I have updated the analyzer CA2208 to support `api_surface` and thought i should add that to this list, maybe it was not needed?

Reply:
I think @Evangelink is suggesting that we should file an issue at https://docs.microsoft.com/visualstudio/code-quality/ca2208 so we update the documentation at https://github.com/MicrosoftDocs/visualstudio-docs-pr/blob/master/docs/code-quality/ca2208.md to mention configurability of API surface. Similar to say this section for CA1000: https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1000?view=vs-2019#configurability

Reply:
Oh I see, thanks @mavasani, updating the doc with CS2208 update is in my plan, I am planning to update the doc myself with all updates I made for the analyzer, not only api_surface change, thanks!

Reply:
Yes that was my intent! Sorry about that.